src/main/java/org/usfirst/frc4048/common/ExampleAdvantageScopeRobot.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.usfirst.frc4048.common;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import org.usfirst.frc4048.common.BuildConstants;
+import org.usfirst.frc4048.common.loggingv2.CommandLogger;
+import org.usfirst.frc4048.common.util.RobotMode;
+import org.littletonrobotics.junction.LogFileUtil;
+import org.littletonrobotics.junction.LoggedRobot;
+import org.littletonrobotics.junction.Logger;
+import org.littletonrobotics.junction.wpilog.WPILOGReader;
+import org.littletonrobotics.junction.wpilog.WPILOGWriter;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class ExampleAdvantageScopeRobot extends LoggedRobot {
+    private static final ConcurrentLinkedQueue<Runnable> runInMainThread = new ConcurrentLinkedQueue<>();
```

Comment:
I agree, it would be better to have the key and the value to log. However, there are so many different loggable types which all call different log methods. It is not impossible, but it feels like work that is not strictly necessary.

Reply:
That's fine. For a followup, I think this whole piece could move into the `CommandLogger` clsss which would solve all issues.